utils/tang/files/tangdw

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
```

Comment:
I really hope this isn't what you need bash for.  (hint, this script doesn't need bash)

Reply:
No I do not need `bash` for this script but for this one: https://github.com/latchset/tang/blob/master/src/tangd-update